### DISCUSSION TITLE
osgEarth 2.10.1 ImageLayer resolution.

### DISCUSSION THREAD

**Discussion by AnthonyDrow21** [pelicanmapping/osgearth#2313] [Support Q&A]
Hello,

I am working on updating a moving map display from osgEarth 2.7 to osgEarth 2.10.1 (I cannot upgrade to osgEarth 3 due to external factors). While I have had to make a few minor API changes, the overall code is relatively the same. I am using the GDALOptions to load a CADRG file into an imageLayer, but no matter how I configure the options I am unable to get the resolution of the image to look good. I have tried setting the minResolution, minLevel, tileSize, and other fields to try and upscale the resolution. I have attached an image detailing what I'm seeing and the code that configures the Image Layer options.

Is this a setting that would be configured in the MapOptions or ProfileOptions instead? Thank you for any assistance. 

```
osgEarth::Drivers::GDALOptions options;
options.url() = filename;

osgEarth::ImageLayerOptions imageOptions(filename, options);
imageOptions.maxLevel() = 100;
imageOptions.tileSize() = 512;

auto layer{new osgEarth::ImageLayer(filename, imageOptions)};
layer->setMaxVisibleRange(layer->getMaxVisibleRange() * 100.0);
map->addLayer(layer);
```
![Moving Map bug](https://github.com/gwaldron/osgearth/assets/12941436/4f81b3a9-3619-4c4f-8411-b3ad10e8c089)

**Comment by gwaldron** [comment]
Hi. It is difficult to say what is wrong without knowing more about the data. Is this a single CARDG? Are you using a VRT? What's the gdalinfo output look like?

**Reply by AnthonyDrow21** [reply]
Thank you for the response! This is a single CADRG map (cgnc) but my repo has more datasets (jnc, onc) and I have noticed a similar problem with the other maps. I am not using a VRT, but I will look into it as it may be a better solution for parsing the files than what is currently done. I am working on producing an output file for gdalinfo, but I have never used this utility before. I am working on getting it built with vcpkg and I will try to add the info here once I get the chance. Are there any specific arguments that ought to be passed to gdalinfo to produce more useful output? I'm very new to these tools and don't want to waste your time while I figure things out. 

I can link the code that scans the map directory as well if you think that would help.

Thank you again for your assistance.
